TLM/TLM/Custom/PathFinding/CustomPathManager.cs

```diff
@@ -273,96 +274,110 @@ public class CustomPathManager : PathManager {
             return false;
         }
 
+        /*internal void ResetQueueItem(uint unit) {
+                queueItems[unit].Reset();
```

Comment:
whats this?

Reply:
Nothing changed here, it's just some really old stuff(>2yo), 363-365 in previous version (Github is not the best in making diffs).

I moved method from 277-361 to the `ExtPathManager` (comment -> 297)